### PR TITLE
[BLOCKS] Accepts in contentset and content selector

### PR DESCRIPTION
**Pull request by cmarianhodis**
Related issue: https://github.com/backbee/backbee-standard/issues/581

### CODE REVIEW

File: src/tb/apps/content/plugins/contentsetplus/main.js [hunk 1]
```diff
@@ -50,18 +51,37 @@ define(
...
-                    this.showPopin();
-                } else {
-                    this.showPopin(this.buildContents(accepts));
-                }
+                var self = this,
+                    content = this.getCurrentContent(),
+                    accepts = content.getDefinition('accept'),
+                    contentType;
+
+                content.getData().done(function () {
```
Comment (fkroockmann) [inline src/tb/apps/content/plugins/contentsetplus/main.js]: You dont need this call, you can just content.getAccept()

File: src/tb/apps/content/plugins/contentsetplus/main.js [hunk 2]
```diff
@@ -50,18 +51,37 @@ define(
...
-                        content.append(newContent);
-                    });
-                } else if (accepts.length === 0) {
-                    this.showPopin();
-                } else {
-                    this.showPopin(this.buildContents(accepts));
-                }
+                var self = this,
+                    content = this.getCurrentContent(),
+                    accepts = content.getDefinition('accept'),
```
Comment (fkroockmann) [inline src/tb/apps/content/plugins/contentsetplus/main.js]: accepts = content.getAccept();

**Comment by fkroockmann** [comment]
I dont understand why you change contentTypeSelector

**Comment by cmarianhodis** [comment]
I made the adjustments and I fixed the content selector as well.

File: src/tb/apps/content/components/ContentManager.js [hunk 3]
```diff
@@ -425,6 +425,26 @@ define(
...
+
+            /**
+             * Replace all occurences of charToReplace with charReplaceWith in an array
+             * 
+             * @param {Array} array
+             * @param {String} charToReplace
+             * @param {String} charReplaceWith
+             * @returns {Array}
+             */
+            replaceChars: function (array, charToReplace, charReplaceWith) {
```
Comment (fkroockmann) [inline src/tb/apps/content/components/ContentManager.js]: array already exist in javascript, can you put other word for less confuse

**Pull request merged**